processing/data_collection/gazette/spiders/rn_mossoro.py

```diff
@@ -0,0 +1,56 @@
+from dateparser import parse
+import datetime as dt
+
+import scrapy
+from scrapy.http import Request
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RnMossoroSpider(BaseGazetteSpider):
+    MUNICIPALITY_ID = '2408003'
+    PDF_URL = 'https://www.prefeiturademossoro.com.br/jom/{}'
+
+    GAZETTES_CSS = 'body > div > table:first-child > tr > td > table:last-child > tr > td > div > table > tr > td'
+    PDF_INFO_CSS = 'td.TextoLink'
+    PDF_REALTIVE_PATH_CSS = 'a::attr(href)'
```

Comment:
Did you mean to spell `relative`? This same spelling repeats along the file.

Reply:
I did. It was a misspelling, sorry about that.